src/angular-business/header/header/header.component.ts

```diff
@@ -218,31 +296,41 @@ export class SbbHeader implements OnInit, AfterViewInit, OnDestroy {
           this.openedChange.emit(this._opened);
         }
       });
+
+    // Programmatically track the collapsed state of the header menus.
+    // This is required in order to attach/detach the menus to the main
+    // or side navigation.
+    this._headerMenusCollapsed = this._collapseBreakpoint.pipe(
+      distinctUntilChanged(),
+      switchMap((breakpoint) => this._breakpointObserver.observe(breakpointMapping[breakpoint])),
+      map((r) => r.matches),
+      distinctUntilChanged(),
+      shareReplay(),
+      takeUntil(this._destroyed)
+    );
   }
 
   ngOnInit() {
     this._checkLabel();
   }
 
+  ngOnChanges(_changes: SimpleChanges): void {
+    this._collapseBreakpoint.next(this.collapseBreakpoint);
```

Comment:
For me a setter for collapseBreakpoint would be clearer

Reply:
This might trigger the ExpressionHasChanged... error